frontend/src/app/spot/components/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,72 @@
+<ng-container
+  *ngFor="let crumb of content.crumbs; let i = index; let last = last; let first = first"
+>
+  <!-- Desktop crumbs -->
+  <div
+    *ngIf="(first && !last) || i === content.crumbs.length - 2 || i === content.crumbs.length - 3 "
+    class="spot-breadcrumbs--crumb hidden-for-mobile"
+    [ngClass]="{'spot-breadcrumbs--crumb_shrink': !first }"
+  >
+    <button
+      type="button"
+      class="spot-link spot-breadcrumbs--crumb-action-item"
+      [title]="crumb.text"
+    >
+      <span *ngIf="crumb.icon" class="spot-icon spot-icon_{{crumb.icon}}"></span>
+      <span [textContent]="crumb.text"></span>
+    </button>
+
+    <span class="spot-breadcrumbs--crumb-divider spot-icon spot-icon_arrow-right5"></span>
+  </div>
+
+  <div
+    *ngIf="i === 1 && content.crumbs.length > 4"
+    class="spot-breadcrumbs--crumb hidden-for-mobile"
```

Comment:
Please write spot-specific rules instead of using the `hidden-for-` utility classes.

Reply:
In fact, I don´t see a mobile/desktop specific version in figma? But I might just be missing it. In any case this template feels like it is way too big for what the component does, though maybe that's warranted.

Reply:
e.g. you can just render one list instead of two and hide specific elements based on classes (e.g. add `spot-breadcrumb--crumb_first` or something, and select based on those)

Reply:
I have the same feeling like you, but...

When I tried to put everything into one list and hide specific elements, the switches between mobile and desktop went crazy, due to the fact, that mobile and desktop just behave soooo different. It is not only display logic, but also shrinkability and content.

In the end I found the `ng-container` with the two sections (desktop items and mobile items) by far the most readable solution. But if you want to pair to find a better solution, please let me know.

Reply:
reworked the template after discussing the possibilities to switch display logic in SASS classes.